mypy/test/testcmdline.py

```diff
@@ -58,6 +58,14 @@ def test_python_cmdline(testcase: DataDrivenTestCase) -> None:
     outb = process.stdout.read()
     # Split output into lines.
     out = [s.rstrip('\n\r') for s in str(outb, 'utf8').splitlines()]
+
+    is_running_in_py_charm = "PYCHARM_HOSTED" in os.environ
```

Comment:
This looks unrelated to your PR.

Reply:
It is slightly unrelated but needed to allow me run tests inside PyCharm, which helps me create the PR. Once the PR is in merge ready state I can pull it out into a different PR.

Reply:
How are you running things from PyCharm? I've been able to run the test suite for months without issue.